Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13 AS builder
```

Comment:
If we want/need to bump we should also bump the associated image in CI and art too

Reply:
Sure, go modules is the default mode in 1.13. I will open follow-up PRs there.

Reply:
Done